src/test/java/org/mockito/internal/creation/bytebuddy/InlineByteBuddyMockMakerTest.java

```diff
@@ -225,6 +225,7 @@ public void is_type_mockable_give_allow_final_mockable_from_JDK() {
     @Test
     public void test_parameters_retention() throws Exception {
         assumeTrue(ClassFileVersion.ofThisVm().isAtLeast(ClassFileVersion.JAVA_V8));
+        assumeTrue(ClassFileVersion.ofThisVm().isLessThan(ClassFileVersion.JAVA_V9)); // Change when ByteBuddy has ASM6
```

Comment:
I see an additional change in the logic here. At least Java 8 vs less than Java 9 (so also Java 6 & 7). Is it an intentional change?